src/Stryker.Core/Stryker.Core/Initialisation/CsharpProjectComponentsBuilder.cs

```diff
@@ -87,12 +87,6 @@ private CsharpFolderComposite FindProjectFilesUsingBuildalyzer(IAnalyzerResult a
 
             foreach (var sourceFile in analyzerResult.SourceFiles)
             {
-                // Skip xamarin UI generated files
-                if (sourceFile.EndsWith(".xaml.cs"))
```

Comment:
Should we actually do anything with these files? Are they not auto generated?

Reply:
No... only `xaml.g.cs` are auto-generated.. and striker already have a way to work with antogen files